app/dashboard/migrations/0093_auto_20200324_1653.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.4 on 2020-03-24 16:53
+
+from django.db import migrations, models
+import django.db.models.deletion
+import economy.models
+
+
+class Migration(migrations.Migration):
```

Comment:
```
CommandError: Conflicting migrations detected; multiple leaf nodes in the migration graph: (0093_auto_20200324_1653, 0095_hackathonevent_visible in dashboard).
To fix them run 'python manage.py makemigrations --merge'
```

Reply:
we are avoiding "merge" on this cases where the migration is on a Pr need to be deleted and recreated.